README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
@kai33 Just added the documentation in latest commit! Thanks for the feedback! :)